hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.jsx

```diff
@@ -61,15 +52,19 @@ export default function ActionRuleFormColumns({
             newClassification.equals = e.target.value === 'true';
             onClassificationChange(newClassification);
           }}>
-          <option key="Equal" value="true">
-            =
+          <option value="true">
+            {classification.classification_type === 'Classification'
+              ? 'Classified As'
```

Comment:
I saw you extrapolated from the reorganization of the logic around "classified as" vs. "is" to conditionally setting the "operator" dropdown. My reflection on that is that it's one thing when you're "writing out" data as english sentences / sentence fragments, but it's another thing when you're having the UI controls for creating that data. I think it's actually "noisy" / possibly confusing to switch between the = and ≠ characters vs. "Classified As" and "Not Classified As". It's considerably wider, too.

Reply:
Reverting